templates/entry/slim.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Slim entry template.
+ *
+ * @package WPAMPTheme
+ */
+
+?>
+<div id="<?php echo esc_attr( get_the_ID() ); ?>-post" data-sort-time="<?php echo esc_attr( get_the_date( 'U' ) ); ?>" class="col-12 sm-col-6 lg-col-4 p1 travel-results-result" style="float:left;">
+	<div class="relative travel-results-result">
+		<a class="travel-results-result-link" href="<?php the_permalink(); ?>">
+			<amp-img class="rounded bg-silver mb1" width="2" height="1" noloading="" layout="responsive" src="<?php echo esc_url( wp_get_attachment_url( get_post_thumbnail_id() ) ); ?>" ></amp-img>
+		</a>
+	</div>
+	<div class="h2 line-height-2 mb1">
+		<span class="travel-results-result-text"><?php echo get_the_title(); ?></span>
+		<span class="travel-results-result-subtext h3">&bull;</span>
+		<span class="travel-results-result-subtext h3">$&nbsp;</span>
+		<span class="bold"><?php echo esc_html( get_post_meta( get_the_ID(), 'amp_travel_price', true ) ); ?></span>
+	</div>
+	<?php get_template_part( 'templates/entry/meta/slim-meta' ); ?>
+</div>
```

Comment:
It looks like the `$` and price sometimes appear on a different line:

<img width="758" alt="styling-product" src="https://user-images.githubusercontent.com/4063887/39015823-1c3c7534-43e4-11e8-9abe-bed4b068f998.png">

The [Firebase demo](https://ampstart-iframes.firebaseapp.com/templates/travel/travel-results.amp) doesn't look to have this issue with the `$`. But it is possible for the `*` and title to be on the same line, like for "Bake Traditional Sweet Bread."